content/syllabuses/short-course-datascience-1.md

```diff
@@ -6,6 +6,25 @@ title: Datascience short course 1
 This course is to be completed in 8 weeks at 10 hours per week. 
 
 Learners doing this course should first do the "Intro to Tilde for tech bootcamps" course.
+# Course Overview
```

Comment:
You are editing the wrong file.

Please look at the rest of the files in the content/syllabuses directory and try to spot the pattern.

The files here are not content files to be displayed to the learners. This is not a topic card. This is not a project card. This is configuration.

This file should contain contentlinks to all of the things that belong inside this syllabus. 

Please make a topic for the course overview. 

content/syllabuses/short-course-datascience-1.md should contain a content link that points to the topic.